src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.ps1

```diff
@@ -611,4 +611,36 @@ function Test-AnalysisServicesServerLoginWithSPN
 	{
 
 	}
+}
+
+<#
+.SYNOPSIS
+Tests Analysis Services server lifecycle  Failure scenarios (Create, Update, Get, Delete).
+#>
+function Test-AnalysisServicesServerGateway
+{
+    try
+    {
+        # Creating server
+        $location = Get-Location
+        $resourceGroupName = Get-ResourceGroupName
+        $serverName = Get-AnalysisServicesServerName
+        $gatewayName = $env:GATEWAY_NAME
+        $gateway = Get-AzureRmResource -ResourceName $gatewayName -ResourceGroupName $resourceGroupName
```

Comment:
So, this test assumes that the gateway server already exists in the subscription/RG... Is it possible to create the gateway server as part of the test?

Otherwise some one (lets say a person in the PS team) would not be able to re-record the test.

Reply:
This have to be a pre-setup azure resource, which requires user to install unified gateway on their on-premise server and generate an gateway azure resource associated on it. It is not possible to automate the procedure in few lines of powershell scripts. This is LiveOnly test.